lib/device-sockets/ios/socket-proxy-factory.ts

```diff
@@ -75,7 +75,7 @@ export class SocketProxyFactory extends EventEmitter implements ISocketProxyFact
 
 	public async createWebSocketProxy(factory: () => Promise<net.Socket>): Promise<ws.Server> {
 		// NOTE: We will try to provide command line options to select ports, at least on the localhost.
-		const localPort = await this.$net.getAvailablePortInRange(8080);
+		const localPort = await this.$net.getFreePort();
```

Comment:
Because it didn't work with multiple iOS devices. For some reason the second device's socket connection could not be established due to an `EADDRINUSE` error. 
It did work for Android however but I decided it was not worth the research at the time - we don't actually lose anything with this change other than the fact that every debug session will be on a random port rather than the first being on 8080, the second on 8081 and so on.